server/src/clients/google.py

```diff
@@ -93,3 +94,87 @@ def get_user_info(
             parsed_user_info[field] = user_info.get(field, "")
 
         return (None, parsed_user_info)
+
+
+@error_bounded(
+    (AppError(ErrorCode.GOOGLE_API_ERROR, "Google api error - get user contacts"), None)
+)
+def _get_api_people_connections_list(
+    token: str,
+    http: Optional[httplib2.Http] = None,
+    http2: Optional[httplib2.Http] = None,
+) -> Tuple[Optional[AppError], Optional[List[Dict[str, Any]]]]:
```

Comment:
It's best to be as specific with the type as you can as avoid 'Any' wherever possible. 
This seems to always return Optional[List[Dict[str,List[str]]]]. Can the 'Any' be anything other than a 'List[str]'?

Reply:
So this is a list of the Google API's "Person" objects, and the Any could be any of: Object, String, List:
https://developers.google.com/people/api/rest/v1/people#Person

wdyt; keep as Any or try to change? @EpicPi